src/applications/personalization/view-dependents/tests/components/ViewDependentsList.unit.spec.jsx

```diff
@@ -58,18 +57,4 @@ describe('<ViewDependentsList />', () => {
     expect(wrapper.find('.loading-indicator-container')).to.exist;
     wrapper.unmount();
   });
-
-  it('Should show a message that the list is empty when no dependents are passed in', () => {
```

Comment:
I don't understand why this test was deleted when it seems like it would still apply.  Can you elaborate?

Reply:
Unit tests will be included in a separate PR